doc/source/release-notes/release-notes_index.rst

```diff
@@ -1,45 +1,82 @@
-   Percona-Server-5.7.17-11
-   Percona-Server-5.7.16-10
-   Percona-Server-5.7.15-9
-   Percona-Server-5.7.14-8
-   Percona-Server-5.7.14-7
-   Percona-Server-5.7.13-6
-   Percona-Server-5.7.12-5
-   Percona-Server-5.7.11-4
-   Percona-Server-5.7.10-3
-   Percona-Server-5.7.10-2
-   Percona-Server-5.7.10-1
+   Percona Server 5.7.35-38 (2021-08-18) <Percona-Server-5.7.35-38.rst>
+
+   Percona Server 5.7.34-37 (2021-05-26) <Percona-Server-5.7.34-37.rst>
+
+   Percona Server 5.7.33-36 (2021-03-02) <Percona-Server-5.7.33-36.rst>
+
+   Percona Server 5.7.32-35 (2020-11-24) <Percona-Server-5.7.32-35.rst>
+
+   Percona Server 5.7.31-34 (2020-08-24) <Percona-Server-5.7.31-34.rst>
+
+   Percona Server 5.7.30-33 (2020-05-20) <Percona-Server-5.7.30-33.rst>
+
+   Percona Server 5.7.29-32 (2020-02-05) <Percona-Server-5.7.29-32.rst>
+
+   Percona Server 5.7.28-31 (2019-11-19) <Percona-Server-5.7.28-31.rst>
+
+   Percona Server 5.7.27-30 (2019-08-22) <Percona-Server-5.7.27-30.rst>
+
+   Percona Server 5.7.26-29 (2019-05-27) <Percona-Server-5.7.26-29.rst>
+
+   Percona Server 5.7.25-28 (2019-02-18) <Percona-Server-5.7.25-28.rst>
+
+   Percona Server 5.7.24-27 (2018-12-18) <Percona-Server-5.7.24-27.rst>
+
+   Percona Server 5.7.24-26 (2018-12-04) <Percona-Server-5.7.24-26.rst>
+
+   Percona Server 5.7.23-25 (2018-11-21) <Percona-Server-5.7.23-25.rst>
+
+   Percona Server 5.7.23-24 (2018-11-09) <Percona-Server-5.7.23-24.rst>
+
+   Percona Server 5.7.23-23 (2018-09-12) <Percona-Server-5.7.23-23.rst>
+
+   Percona Server 5.7.22-22 (2018-04-19) <Percona-Server-5.7.22-22.rst>
+
+   Percona Server 5.7.21-21 (2018-04-24) <Percona-Server-5.7.21-21.rst>
+
+   Percona Server 5.7.21-20 (2019-02-18) <Percona-Server-5.7.21-20.rst>
+
+   Percona Server 5.7.20-19 (2018-01-03) <Percona-Server-5.7.20-19.rst>
+
+   Percona Server 5.7.20-18 (2017-12-14) <Percona-Server-5.7.20-18.rst>
+
+   Percona Server 5.7.19-17 (2017-08-31) <Percona-Server-5.7.19-17.rst>
+
+   Percona Server 5.7.18-16 (2017-07-28) <Percona-Server-5.7.18-16.rst>
+
+   Percona Server 5.7.18-15 (2017-05-26) <Percona-Server-5.7.18-15.rst>
+
+   Percona Server 5.7.18-14 (2017-05-12) <Percona-Server-5.7.18-14.rst>
+
+   Percona Server 5.7.17-13 (2017-04-05) <Percona-Server-5.7.17-13.rst>
+
+   Percona Server 5.7.17-12 (2017-03-24) <Percona-Server-5.7.17-12.rst>
+
+   Percona Server 5.7.17-11 (2017-02-03) <Percona-Server-5.7.17-11.rst>
+
+   Percona Server 5.7.16-10 (2016-11-28) <Percona-Server-5.7.16-10.rst>
+
+   Percona Server 5.7.15-9 (2016-10-21) <Percona-Server-5.7.15-9.rst>
+
+   Percona Server 5.7.14-8 (2016-09-21) <Percona-Server-5.7.14-8.rst>
+
+   Percona Server 5.7.14-7 (2016-08-23) <Percona-Server-5.7.14-7.rst>
+
+   Percona Server 5.7.13-6 (2016-07-16) <Percona-Server-5.7.13-6.rst>
+
+   Percona Server 5.7.12-5 (2016-06-06) <Percona-Server-5.7.12-5.rst>
+
+   Percona Server 5.7.11-4 (2016-03-16) <Percona-Server-5.7.11-4.rst>
+
+   Percona Server 5.7.10-3 (2016-02-16) <Percona-Server-5.7.10-3.rst>
```

Comment:
Percona Server for MySQL 5.7.10-3 on February 23rd, 2016

Reply:
Fixed